### PR TITLE
Fix encoding bug

### PR DESCRIPTION
**Pull request by lyriccoder**
fixing #324

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#325](https://github.com/yegor256/aibolit/issues/325) doesn't exist, can't set role

File: aibolit/patterns/string_concat/string_concat.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def value(self, filename: str) -> List[LineNumber]:
                 is_l_this = isinstance(node.operandl, javalang.tree.This)
                 is_r_this = isinstance(node.operandr, javalang.tree.This)
                 if is_l_literal and (is_r_member or is_r_meth_inv or is_r_this):
-                    is_string_literal = '"' in node.operandl.value
+                    is_string_literal = '"' in node.operandl.value  # type: ignore
```
Comment (acheshkov) [inline aibolit/patterns/string_concat/string_concat.py:31]: why type ignore?

Comment (lyriccoder) [reply aibolit/patterns/string_concat/string_concat.py:31]: because mypy thinks that it doesn't have such a value

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/aibolit#325` is not assigned, can't get performer

**Pull request merged**